server.js

```diff
@@ -218,11 +218,33 @@ app.get('/live/:year/:week', function(req, res) {
 	});
 })
 
+/**
+ * GET route for the current week in the season. This route can be used when
+ * wanting to know the current season and the current week in the season.
+ */
+ app.get('/currentSeasonAndWeek', function(req, res) {
```

Comment:
I think this might be better written out as `/current` or something shorter than what you've implemented.